bundle/language_es.properties

```diff
@@ -1175,6 +1175,8 @@ angal.patient.parenttogether
 angal.patient.patientbrowser.title                                                                     = Lista Pacientes
 angal.patient.patientphoto                                                                             = Foto Paciente
 angal.patient.patientselection.title                                                                   = Seleccionar Paciente
+angal.patient.patientphoto.error.title                                                                 = Error en la foto del paciente
+angal.patient.patientphoto.error.filenotfound                                                          = No se encontr� la foto del paciente.
```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.